src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -190,11 +197,22 @@ AsyncReplResult<> RaftReplDev::replace_member(replica_id_t member_out_uuid, repl
                     } else {
                         RD_LOGI("Replace member removed member={} from group_id={}", member_out, group_id_str());
                     }
+
+                    // Revert the quorum size back to 0.
+                    reset_quorum_size(0);
                     return make_async_success<>();
                 });
         });
 }
 
+void RaftReplDev::reset_quorum_size(uint32_t commit_quorum) {
```

Comment:
do we need add some check here to disable setting those quorum sizes without intersection, since it might case  data loss or log diverging.
pls see 
https://github.com/eBay/NuRaft/blob/092997ab67eb2529c5b40556e1d301c8220c315d/docs/custom_quorum_size.md?plain=1#L29

Reply:
this is used when we have double failure , e.g ( 2 out of 3 members are down )--- so for sure the number *seems* wrong.  

We need other mechanism to shot on the head of other members , ensuring they are deadly dead.

Reply:
when we make this config change, there might be some on-going write or delete, we need to carefully handle this quorum size change.

Reply:
We need to make these PG's read only as two nodes are down. Will check with CM, in nublocks once removed  they are never added back to cluster.